deploy/olm-catalog/ibm-block-csi-operator/1.8.0/metadata/annotations.yaml

```diff
@@ -5,3 +5,5 @@ annotations:
   operators.operatorframework.io.bundle.mediatype.v1: registry+v1
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: ibm-block-csi-operator
+  com.redhat.openshift.versions: "v4.7-v4.9"
+  com.redhat.delivery.operator.bundle: true
```

Comment:
looks like a newline is missing
![image](https://user-images.githubusercontent.com/57443811/146210521-b83985ab-c7c6-4e44-8e6c-755e155a23d9.png)

Reply:
@lisyukanna , apparently few more changes are required here:
1. remove the quote marks from openshift version part "v4.7-v4.9"
2. remove the last line, it is not needed